integration/command.go

```diff
@@ -69,8 +69,9 @@ var deployInspektorGadget *command = &command{
 }
 
 var waitUntilInspektorGadgetPodsDeployed *command = &command{
-	name: "Wait until the gadget pods are started",
-	cmd:  "for POD in $(sleep 5 ; kubectl get pod -n gadget -l k8s-app=gadget -o name) ; do kubectl wait --timeout=30s -n gadget --for=condition=ready $POD ; done ; kubectl get pod -n gadget",
+	name:           "Wait until the gadget pods are started",
+	cmd:            "ret=\"\" ; for POD in $(sleep 5 ; kubectl get pod -n gadget -l k8s-app=gadget -o name) ; do kubectl wait --timeout=30s -n gadget --for=condition=ready $POD; ret=\"$ret$(echo $?)\" ; done ; kubectl get pod -n gadget ; echo \"ret=$ret\" ; ",
+	expectedRegexp: "\nret=[0]+\n",
```

Comment:
@mauriciovasquezbernal I agree but I am not sure if people would like to always wait, maybe we could add a flag to disable it 🤔? I was thinking the same for the `undeploy` command, shouldn't it wait until the `gadget` namespace is removed? Every time I need to redeploy the IG I always execute: 

```
kubectl gadget undeploy; kubectl wait --for=delete namespace gadget --timeout=-1s
```

Maybe that's a special case because I am redeploying and not just undeploying 🤔 . 

Anyway, that will be managed in another PR, I updated this one with your suggestion, I liked it.

Reply:
> shouldn't it wait until the gadget namespace is removed?

Yeah, it's something I have on my TODO list. I also hit the same problem when trying to redeploy.

Reply:
> I agree but I am not sure if people would like to always wait, maybe we could add a flag to disable it thinking?

I think the default should be to wait, if you run `kubectl gadget deploy` you'd expect that just after it you can run `kubectl gadget <do something>`. Actually TBH I don't see in which use case not waiting will be useful :thinking: 

I think the command could be print useful information to the user while it's installing, so they know what's going on:

```
$ kubectl gadget deploy 
Deploying Inspektor Gadget vXXXX in N nodes 
Pulling image ...
x/N nodes ready ...
....
Inspektor gadget was deployed successfully
```